### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Ayushparikh-code**
fixes some typo mistakes

### CODE REVIEW

**Comment by Ayushparikh-code** [comment]
> @googlebot I signed it!

@googlebot I signed it!

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## Summary
 
 Turbinia is an open-source framework for deploying, managing, and running
-distributed forensic workloads. It is intended to automate running of common
```
Comment (hacktobeer) [inline README.md:6]: removing "running of" completly makes it more natural.

File: README.md [hunk 2]
```diff
@@ -13,13 +13,13 @@ possible.
 
 ## How it works
 
-Turbinia is composed of different components for the client, server and the
-workers. These components can be run in the Cloud, on local machines, or as a
-hybrid of both. The Turbinia client makes requests to process evidence to the
-Turbinia server. The Turbinia server creates logical jobs from these incoming
-user requests, which creates and schedules forensic processing tasks to be run
+Turbinia is composed of different components for the client, server, and workers. 
+These components can be run in the Cloud, on local machines, or as a hybrid of both.
```
Comment (hacktobeer) [inline README.md:17]: adding new lines has no impact on markdown as it will still concatenate all lines.

**Review by hacktobeer — COMMENTED** [review]
Thanks for the contribution, couple of small nits.

**Comment by Ayushparikh-code** [comment]
> Thanks for the contribution, couple of small nits.

Ty , review It again plz

**Review by Ayushparikh-code — COMMENTED** [review]
👍

**Comment by hacktobeer** [comment]
Hi, you resolved the comments but did not make changes, correct?

**Review by 5alidshammout — APPROVED** [review]

**Comment by hacktobeer** [comment]
Thanks both. I am going to close this PR as the changes are not relevant and impactful enough. Feel free to have a look at the open issues and pick one of those. Happy to review a more substanstial PR.

**Pull request closed without merging**